docs/src/further_topics/ugrid/operations.rst

```diff
@@ -0,0 +1,791 @@
+.. _ugrid operations:
+
+Working with Mesh Data
+***********************
+
+.. note:: Several of the operations below rely on the optional dependencies
+          mentioned in :doc:`partner_packages`.
+
```

Comment:
Title? "Operation summary"? Would make it easier to link to when people ask us about them too

Reply:
I think on balance I'd like to keep `Working with Mesh Data`.

Reply:
Oh, sorry, I meant add an extra subtitle for this table so I know what I'm looking at (under the `Working with Mesh Data` heading).

Reply:
ef47b6b7d0aad7ec7e5b28ea5375f1cfae9db106